tests/integration/features/istio/production/main-suite/istio_observability.feature

```diff
@@ -0,0 +1,50 @@
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "connection_termination_details"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "upstream_transport_failure_reason"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "bytes_received"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "bytes_sent"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "duration"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "upstream_service_time"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "x_forwarded_for"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "user_agent"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "request_id"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "authority"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "upstream_host"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "upstream_cluster"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "upstream_local_address"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "downstream_local_address"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "downstream_remote_address"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "requested_server_name"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "route_name"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "traceparent"
+    And Envoy logs of deployment "istio-ingressgateway" in namespace "istio-system" contains access log entry with "tracestate"
+
+  Scenario: Istio interacts with the otel collector service defined in meshconfig
+    Given Logging and tracing is enabled for the mesh using providers "stdout-json" for logs and "kyma-traces" for traces
+    And Httpbin application "otel-collector-mock" is running in namespace "kyma-system"
+    And Service is created for the otel collector "otel-collector" in namespace "kyma-system"
```

Comment:
It's not a name of the service, because it is hardcoded in the meshconfig, this is a selector for matching Service to the Deployment, I'll refactor to make it a bit clear so the selector's value will be the same as deployment name